### DISCUSSION TITLE
SC 1.4.1 Use of color and links

### DISCUSSION THREAD

**Discussion by sarasuri** [w3c/wcag#4467] [Q&A] [answered]
Hello,

We have a question regarding the SC 1.4.1 Use of color and links. 

On a web page, a link is distinguished by a different color and bold text within a block of content. However, a regular text within the same block is also bolded. The color contrast between the link and the surrounding text is below 3:1 (1.9:1). In this scenario, would this be considered a failure meeting the Success Criterion 1.4.1? The insufficient contrast and use of bold text for both links and regular text may make it difficult for some users to clearly distinguish between the two.

![image](https://github.com/user-attachments/assets/3b556289-ec6f-48a9-bb53-f1d3350eeb17)

We would appreciate any feedback on this scenario.
Thank you!

**Comment by TestPartners** [comment, marked as answer]
It's a bit harsh in this specific case, but I would say it's a non-conformance.

**Comment by detlevhfischer** [comment]
I think you would need to look at the contrast difference between bold text bits and bold link text (because the difference between bold links and non-bold surrounding text would be clear by virtue of the bold style) -- and that contrast seems to be around 3:1 depending on which pixel you get... so maybe not a FAIL.

**Comment by sarasuri** [comment]
Thank you @TestPartners and @detlevhfischer for your quick responses.

@detlevhfischer, The contrast ratio between the bolded text and the bolded link is 1.9: 1.
Bolded regular text = #424242
Non bolded regular text = #424242
Bolded link = #436e95

**Comment by detlevhfischer** [comment]
@sarasuri ah ok, the dangers of pixel-picking. In that case I guess Steve is right, even though (depending on the page context) the issue seems minor.

**Comment by bruce-usab** [comment]
> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.

Well, not a fail of 1.4.1 since emphasized text is also used to convey meaning.

> The visual [presentation](https://www.w3.org/TR/WCAG22/#dfn-presentation) of the following have a [contrast ratio](https://www.w3.org/TR/WCAG22/#dfn-contrast-ratio) of at least 3:1 against adjacent color(s)...

The link is a UIC and dark blue against white has contrast, so not a fail of 1.4.11.

I agree that it is a problem, and a disproportionate barrier for people with low vision, but I don't think it fails any 2.x SC.  It is the sort of thing I would flag in an audit.  It is also the kind of thing we would want WCAG 3 to cover.

**Comment by patrickhlauke** [comment]
> Well, not a fail of 1.4.1 since emphasized text is also used to convey meaning.

but that's the point. in the example, bold is used both to emphasise static text in that example, AND to denote links. the only other thing that then distinguishes links from static emphasised text is color, so this *would* fail 1.4.1
